         
时间： 2021-12-14 23:01:21.8162         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/14 23:01:21
Application UnhandledException:String 引用没有设置为 String 的实例。
参数名: s;
堆栈信息:   在 System.Text.Encoding.GetBytes(String s)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 361
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.ClientAuthenticationAndReconnection() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 250
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SocketClientReceive() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 228
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()         
堆栈：  Encoding.GetBytes => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-14 23:02:42.5118         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息： System.ArgumentNullException: String 引用没有设置为 String 的实例。
参数名: s
   在 System.Text.Encoding.GetBytes(String s)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 363         
堆栈：  SocketClientManager.ClientAuthenticationAndReconnection => SocketClientManager.SendMsg => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-14 23:03:50.0110         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/14 23:03:49
Application UnhandledException:执行命令定义时出错。有关详细信息，请参阅内部异常。;
堆栈信息:   在 System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   在 System.Data.Entity.Core.Objects.Internal.ObjectQueryExecutionPlan.Execute[TResultType](ObjectContext context, ObjectParameterCollection parameterValues)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass12.<BatchRefreshEntitiesByKey>b__11()
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass12.<BatchRefreshEntitiesByKey>b__10()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.BatchRefreshEntitiesByKey(RefreshMode refreshMode, Dictionary`2 trackedEntities, EntitySet targetSet, List`1 targetKeys, Int32 startFrom)
   在 System.Data.Entity.Core.Objects.ObjectContext.RefreshEntities(RefreshMode refreshMode, IEnumerable collection)
   在 System.Data.Entity.Core.Objects.ObjectContext.Refresh(RefreshMode refreshMode, Object entity)
   在 System.Data.Entity.Internal.InternalEntityEntry.Reload()
   在 System.Data.Entity.Infrastructure.DbEntityEntry`1.Reload()
   在 NanXingPengMaServices.Utils.InOutThread.Init() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\InOutThread.cs:行号 31
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()         
堆栈：  DefaultSqlExecutionStrategy.Execute => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-14 23:03:50.0120         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息：  出现应用程序未处理的异常：2021/12/14 23:03:49
Application UnhandledException:执行命令定义时出错。有关详细信息，请参阅内部异常。;
堆栈信息:   在 System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   在 System.Data.Entity.Core.Objects.Internal.ObjectQueryExecutionPlan.Execute[TResultType](ObjectContext context, ObjectParameterCollection parameterValues)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass12.<BatchRefreshEntitiesByKey>b__11()
   在 System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   在 System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass12.<BatchRefreshEntitiesByKey>b__10()
   在 System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   在 System.Data.Entity.Core.Objects.ObjectContext.BatchRefreshEntitiesByKey(RefreshMode refreshMode, Dictionary`2 trackedEntities, EntitySet targetSet, List`1 targetKeys, Int32 startFrom)
   在 System.Data.Entity.Core.Objects.ObjectContext.RefreshEntities(RefreshMode refreshMode, IEnumerable collection)
   在 System.Data.Entity.Core.Objects.ObjectContext.Refresh(RefreshMode refreshMode, Object entity)
   在 System.Data.Entity.Internal.InternalEntityEntry.Reload()
   在 System.Data.Entity.Infrastructure.DbEntityEntry`1.Reload()
   在 NanXingPengMaServices.Utils.AGVMissionThread.Init() 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\AGVMissionThread.cs:行号 28
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart()         
堆栈：  DefaultSqlExecutionStrategy.Execute => Program.CurrentDomain_UnhandledException => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-14 23:04:46.9150         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息： System.ArgumentNullException: String 引用没有设置为 String 的实例。
参数名: s
   在 System.Text.Encoding.GetBytes(String s)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 363         
堆栈：  SocketClientManager.ClientAuthenticationAndReconnection => SocketClientManager.SendMsg => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-14 23:10:11.7339         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息： System.Net.Sockets.SocketException (0x80004005): 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Send(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   在 System.Net.Sockets.Socket.Send(Byte[] buffer)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg) 位置 F:\项目\南兴果仁\南兴果仁WMS智能仓储管理系统\NanXingPengMaServices\Utils\TCP\ClientBySocket.cs:行号 364         
堆栈：  TiShengJiHelper.WriteToTcp => SocketClientManager.SendMsg => Logger.Process         

-----------------------------------------------------------
         
时间： 2021-12-14 23:18:48.3150         
来源： NanXingPengMaServices.Logger.Process         
等级： Warn         
信息： System.Net.Sockets.SocketException (0x80004005): 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Send(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   在 System.Net.Sockets.Socket.Send(Byte[] buffer)
   在 NanXingPengMaServices.Utils.TCP.SocketClientManager.SendMsg(String msg)         
堆栈：  TiShengJiHelper.WriteToTcp => SocketClientManager.SendMsg => Logger.Process         

-----------------------------------------------------------
